src/main/java/app/PostController.java

```diff
@@ -0,0 +1,90 @@
+package app;
+
+
+import org.springframework.http.*;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+@RestController
+public class PostController {
+
+    @GetMapping("/")
+    public String elo() {
+        return "ELO";
+    }
+
+
+    @GetMapping(path = "/api/posts")
+    public void getPosts() {
```

Comment:
hmmm, what do we have here? `GET` method in restController which return `void` - so does not return anything... Hmm... so what someone who called this method will GET? :D

Reply:
Made it return List<PostDto>